### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by marinasugita**
Check formatting, naming, and logic @JoshuaKnewstub

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
```
Comment (bottle139) [inline prac_05/state_names.py]: TODO has been completed

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
```
Comment (bottle139) [inline prac_05/state_names.py:4]: File has been reformated

File: prac_05/state_names.py [hunk 3]
```diff
@@ -4,15 +4,18 @@
...
 while state != "":
     if state in STATE_NAMES:
         print(state, "is", STATE_NAMES[state])
     else:
         print("Invalid short state")
-    state = input("Enter short state: ")
+    state = input("Enter short state: ").upper()
+
+for state, value in STATE_NAMES.items():
+    print("{:<4} is {}".format(state, value))
```
Comment (bottle139) [inline prac_05/state_names.py]: Should have newline at end of file for PEP8

File: prac_05/colour_names.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+CP1404/CP5632 Practical
+Colour names in a dictionary
+"""
+
+COLOUR_NAMES = {"burlywood": "#deb887", "chocolate": "#d2691e", "coral": "#ff7f50", "pink": "#ffc0cb", "grey": "#bebebe", "plum": "#dda0dd", "khaki": "#f40e68c", "lavender": "#e6e6fa", "maroon": "#b03060", "orchid": "da70d6"}
+
+colour = input("Enter colour: ").lower()
+while colour != "":
+    if colour in COLOUR_NAMES:
+        print(colour, "is", COLOUR_NAMES[colour])
```
Comment (bottle139) [inline prac_05/colour_names.py:11]: Could capitalise the colour name to make neater

File: prac_05/state_names.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
```
Comment (bottle139) [inline prac_05/state_names.py:8]: TODO completed

File: prac_05/word_count.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+words = input("Text: ").split()
```
Comment (bottle139) [inline prac_05/word_count.py:1]: "Text" does not explain what the user should input

File: prac_05/state_names.py (hunk 1 above)
Comment (marinasugita) [reply prac_05/state_names.py]: I've now removed TODO.

File: prac_05/state_names.py (hunk 2 above)
Comment (marinasugita) [reply prac_05/state_names.py:4]: This line of comment has now been removed.

File: prac_05/state_names.py (hunk 3 above)
Comment (marinasugita) [reply prac_05/state_names.py]: Done.

**Pull request merged**